### PR TITLE
Add aspects to curriculum/javascript/core/recipes-ii/

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/recipes-ii/milliseconds-since-epoch.md?ref=1968b4e5d62075804c14d326812ed43032fcbae9

File: javascript/core/recipes-ii/counting-the-months-from-zero.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ tags:
 
 aspects:
   - workout
+  - deep
```
Comment (ryanwarsaw) [inline javascript/core/recipes-ii/counting-the-months-from-zero.md]: This is obscura, but definitely not deep.

File: javascript/core/recipes-ii/js-practice-date.md [hunk 2]
```diff
@@ -10,12 +10,17 @@ levels:
...
+
 linkType: codewars
+
 standards:
 
   javascript.standard-library.5: 1000
   javascript.write-expressions.0: 1000
 
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/recipes-ii/js-practice-date.md]: This is `introduction`

File: javascript/core/recipes-ii/milliseconds-since-epoch.md [hunk 3]
```diff
@@ -13,6 +13,10 @@ standards:
 
   javascript.standard-library.5: 10
 
+aspects:
+  - deep
```
Comment (ryanwarsaw) [inline javascript/core/recipes-ii/milliseconds-since-epoch.md]: Not `deep`, I would consider this `introduction` and `workout`.

**Pull request merged**